lib/matplotlib/backends/backend_qt5.py

```diff
@@ -415,9 +415,6 @@ def stop_event_loop(self):
 
     stop_event_loop.__doc__ = FigureCanvasBase.stop_event_loop_default.__doc__
 
-    def draw_idle(self):
```

Comment:
Its not really clear to me what's the idea of the split between qt5 and qt5agg. Draw is defined in qt5agg, draw_idle is defined in qt5. It seems reasonable to have both at the same spot where the whole draw mechanism is implemented. Its not gone, it just moved from qt5 to qt5agg.